portfolio/src/main/webapp/script.js

```diff
@@ -13,10 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds "Hello Naa'Oli!" to the page.
+ * Adds greetings to the page.
  */
 function getHelloName() {
-
-  const helloNameContainer = document.getElementById('container');
-  helloNameContainer.innerText = "Hello Naa'Oli!";
+  fetch('/data').then(response => response.json()).then((message) => {
+    document.getElementById('container').innerText = message;
+  });
 }
+
+async function getHelloNameAsync() {
```

Comment:
Maybe just keep this function and delete the other one (if it will never be used)? And you can change function name to getHelloName.